### PR TITLE
Adding guidance to not hard wrap files

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: https://github.com/openshift/openshift-docs/blob/8aa0481d1299e1cf495886b4f560aa195771c71c/contributing_to_docs/doc_guidelines.adoc#general-file-guidelines

### CODE REVIEW

**Comment by bergerhoffer** [comment]
FYI @openshift/team-documentation

**Review by kalexand-rh — COMMENTED** [review]
LGTM

We might want to open another discussion about what to say about existing text that has hardwraps and when to unwrap it.

**Comment by bergerhoffer** [comment]
@kalexand-rh Good point. I could add something like it's not necessary to fix existing content, but that you can if you want to. Wdyt?

**Comment by kalexand-rh** [comment]
> @kalexand-rh Good point. I could add something like it's not necessary to fix existing content, but that you can if you want to. Wdyt?

I think that would work, but I think it might be better to ask that people limit their unwrapping activity to sections of the topics that they're actively changing to help keep a useful Git blame.

**Comment by bergerhoffer** [comment]
> I think that would work, but I think it might be better to ask that people limit their optional unwrapping activity to sections of the topics that they're actively changing to help keep a useful Git blame.

Fair enough. How about I update to say something like it's not necessary to update existing content to unwrap, but if you are fixing content you are working in, here's the Atom shortcut, etc.

**Comment by bergerhoffer** [comment]
@kalexand-rh Updated, if you don't mind taking another look to see if what I added conveys what you were looking for. I know that "feel free" is a little unofficial, so feel free to suggest an alternative :laughing:

File: contributing_to_docs/doc_guidelines.adoc [hunk 1]
```diff
@@ -11,6 +11,14 @@ toc::[]
 == General file guidelines
 
 * Set your editor to strip trailing whitespace.
+* Do *not* hard wrap lines at 80 characters (or at any other length).
++
+It is not necessary to update existing content to unwrap lines, but you can feel free to fix any lines that you are currently working in.
```
Comment (kalexand-rh) [inline contributing_to_docs/doc_guidelines.adoc:16]: Maybe

```suggestion
It is not necessary to update existing content to unwrap lines, but you can remove existing wrapping from any lines that you are currently working in.
```

**Comment by bergerhoffer** [comment]
@kalexand-rh Thanks, updated.

**Review by kalexand-rh — COMMENTED** [review]
It's beautiful. 👍

**Pull request merged**